bundles/org.openhab.binding.keba/src/main/resources/OH-INF/thing/kecontact.xml

```diff
@@ -134,82 +134,82 @@
 		<state readOnly="false"></state>
 	</channel-type>
 	<channel-type id="current_settable">
-		<item-type>Number</item-type>
+		<item-type>Number:ElectricCurrent</item-type>
 		<label>Preset Current</label>
-		<description>Preset Current in mA</description>
-		<state pattern="%d mA" readOnly="false"></state>
+		<description>Preset Current in A</description>
+		<state pattern="%.3f A" readOnly="false"></state>
```

Comment:
The unit shouldn't be mentioned in the description as the user can choose an arbitrary unit. You can use the placeholder in the pattern.
```suggestion
		<description>Preset Current</description>
		<state pattern="%.3f %unit%" readOnly="false" />
```

Reply:
done